### ISSUE TITLE
The ce_loss became negative when I was using the mask2former to do instance segmentation

### ISSUE THREAD

**Issue by AIzealotwu** [facebookresearch/detectron2#5300]
command you run:
python train_net.py --num-gpus 8 --config-file configs/coco/instance-segmentation/swi
n/maskformer2_swin_large_IN21k_384_bs16_100ep.yaml MODEL.WEIGHTS "weig
hts/model.pkl"

The data form used in my case is coco.
And the registration of this dataset was strictly followed with the document of our office

**Comment by Programmer-RD-AI** [comment]
Hi,
To resolve the issue you are facing, it is usually a case in the training step..
Try and change the LR (learning rate) and adjust your training process and check if it fixes the issue.
Thank you

**Comment by AIzealotwu** [comment]
Well, Thanks. Maybe this issue was caused by so large of my picture. The learning rate was followed with the default in the mask2former source paper.

**Comment by Programmer-RD-AI** [comment]
Hi,
Glad you found it useful! :)
Try adjusting parameters related to image resolution, batch size, augmentation techniques, or even model architecture to fine-tune your instance segmentation setup. 
If you need further guidance or have specific questions, feel free to ask!
